migrations/20200721091300-encrypt-tokens.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import models, { Op } from '../server/models';
+import { crypto } from '../server/lib/encryption';
+
+module.exports = {
+  up: async queryInterface => {
+    let lastId = 0,
+      count = 0,
+      total = 0;
+    console.info('Encrypting ConnectedAccounts.');
+    do {
+      const accounts = await models.ConnectedAccount.findAll({
```

Comment:
Using the model to retrieve/edit data in migrations is dangerous because it may be out of sync with the DB. Here is why:

<details>
1. You create a model like `Animal { name }`
2. You create a migration with `models.Animal.findAll(...)` in it.
3. You add a `type` column to Animal

When (2) is going to run for the first time (while (3) doesn't exist yet) your SQL query is going to look like this:

```sql
SELECT name FROM animal
```

No problem here. **But** when future contributors will run the migrations from scratch after (3) has been added to the codebase, the query from (2) will look like this:

```sql
SELECT name, type FROM animal
```

This is going to crash because `type` has not been created yet. 

The rule is: we shouldn't assume that the model and the database are synchronized when used in transactions (the goal of transactions being precisely to reconciliate the database with the model).
</details>

To fix it, you can either:
- Specify `attributes`, to make sure we only fetch the required attributes
- Or use raw SQL

Reply:
Alternative, move the file to the `scripts` folder and run it manually.